.github/workflows/docker-check.yaml

```
@@ -0,0 +1,82 @@
+    name: Docker Check
+    runs-on: ubuntu-latest
+    if: github.event.issue.pull_request && contains(github.event.comment.body, '/cmd docker --check')
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ github.event.issue.pull_request.head.sha }}
+          repository: ${{ github.event.repository.full_name }}
+      - name: Post Initial Status
+        if: github.event_name == 'issue_comment'
+        uses: peter-evans/create-or-update-comment@v3
```

actionlint (1.7.9)

the runner of "peter-evans/create-or-update-comment@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)